doc/how-to/initialise.rst

```diff
@@ -90,6 +90,39 @@ Complete the following steps to initialise MicroCloud:
 
 See an example of the full initialisation process in the :ref:`Get started with MicroCloud <initialisation-process>` tutorial.
 
+Excluding MicroCeph or MicroOVN from MicroCloud
+-----------------------------------------------
```

Comment:
```suggestion
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```

This section should be a subsection of "Interactive configuration", right?
Then we'll need to use another symbol for the heading (not sure about the length of the suggestion :wink: - should be as long as the title).